app/src/main/res/values/colors.xml

```diff
@@ -12,5 +12,6 @@
     <color name="whitesmoke">#FFF5F5F5</color>
     <color name="silverchalice">#FFB0B0B0</color>
     <color name="red_orange">#FF2424</color>
+    <color name="disbaled_icon">#8E8888</color>
```

Comment:
refactor this to something like `remindi_disabled_icon`

Reply:
also can you do me a slight favour?
I have accidentally named the remindi colors in upperCamelCase, can you just refactor them to snake_case in the next commit?
Thank you in advance!

Reply:
Yes I too noticed it...will do the needful